README.md

```diff
@@ -341,6 +362,117 @@ DELETE `/v1/s3/{account}/buckets/{bucket}/users/{user}
 | **429 Too Many Requests**     | service or rate limit exceeded           |  
 | **500 Internal Server Error** | a server error occurred                  |
 
+### Create a website
+
+POST `/v1/s3/{account}/websites
+
+#### Request
+
+```json
+{
+    "Tags": [
+        { "Key": "Application", "Value": "HowToGet" },
+        { "Key": "COA" "Value", "Value": "Take.My.Money.$$$$" },
+        { "Key": "CreatedBy", "Value": "Big Bird" }
+    ],
+    "BucketInput": {
+        "Bucket": "foobar.bulldogs.cloud"
+    }
+}
+```
+
+#### Response
+
+```json
+{
+    "Bucket": "/foobar.bulldogs.cloud",
+    "Policy": {
+        "Arn": "arn:aws:iam::846761448161:policy/foobar.bulldogs.cloud-BktAdmPlc",
+        "AttachmentCount": 0,
+        "CreateDate": "2019-03-01T15:33:52Z",
+        "DefaultVersionId": "v1",
+        "Description": null,
+        "IsAttachable": true,
+        "Path": "/",
+        "PermissionsBoundaryUsageCount": 0,
+        "PolicyId": "ABCDEFGHI12345678",
+        "PolicyName": "foobar.bulldogs.cloud-BktAdmPlc",
```

Comment:
What's the length limit for `PolicyName` and `GroupName`?

Reply:
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-limits.html#reference_iam-limits-entity-length

Reply:
ok, 128 should be plenty